test/e2e/apps/statefulset.go

```diff
@@ -125,7 +125,7 @@ var _ = SIGDescribe("StatefulSet", func() {
 
 		// This can't be Conformance yet because it depends on a default
 		// StorageClass and a dynamic provisioner.
-		ginkgo.It("should provide basic identity", func() {
+		ginkgo.It("should provide basic identity [feature: CloudProvider]", func() {
```

Comment:
The convention is to use "[Feature: x]" so that our CI jobs will automatically skip those tests.

Reply:
Thanks! replaced all with upper case.